components/CommentModal/PostSubmitModal.js

```diff
@@ -0,0 +1,128 @@
+import React from "react"
+import { siteUrl } from "../links"
+import { Button, Col, Modal } from "../bootstrap"
+import { useMember, useProfile } from "../db"
+
+import EmailToMyLegislatorsComponent from "./EmailToMyLegislatorsComponent"
+import EmailToCommitteeComponent from "./EmailToCommitteeComponent"
+import TweetComponent from "./TweetComponent"
+
+import createCommitteeChairEmailCommand from "./createCommitteeChairEmailCommand"
+import createMyLegislatorEmailCommand from "./createMyLegislatorEmailCommand"
+
+const PostSubmitModal = ({
+  showPostSubmitModal,
+  setShowPostSubmitModal,
+  handleCloseTestimony,
+  bill,
+  testimony,
+  senateChairEmail,
+  houseChairEmail,
+  committeeName
+}) => {
+  const billNumber = bill?.BillNumber
+  const billTitle = bill?.Title
+  const testimonyContent = testimony?.content
+
+  const { profile } = useProfile()
+  const senator = useMember(profile?.senator?.id)
+  const representative = useMember(profile?.representative?.id)
+  const senatorEmail = senator.member?.EmailAddress ?? ""
+  const representativeEmail = representative.member?.EmailAddress ?? ""
+
+  const testimonyArchiveEmailAddress = "test@example.com" // in order to have emails send to legislators via BCC, we need a primary "send to" email address for each email.  This is a placeholder email address.  Ultimately, this should be in a configuration file.
```

Comment:
For the beta, we won't have an email address for this. Can we remove it for now and move the legislators to the to field?

Reply:
ok.  I changed the emails back to being sent directly to the legislators instead of bcc.